config/routes.rb

```diff
@@ -1,56 +1,29 @@
 Rails.application.routes.draw do
-  # The priority is based upon order of creation: first created -> highest priority.
-  # See how all your routes lay out with "rake routes".
```

Comment:
Thank you so much Wes!
-Leah

On Tue, May 24, 2016 at 8:44 PM, wes <notifications@github.com> wrote:

> In config/routes.rb
> <https://github.com/Ada-C5/tunes-takeout/pull/1#discussion_r64511396>:
>
> > @@ -1,56 +1,29 @@
> >  Rails.application.routes.draw do
> > -  # The priority is based upon order of creation: first created -> highest priority.
> > -  # See how all your routes lay out with "rake routes".
>
> Way to delete the tons of boilerplate code that comes in routes.rb. Also
> good comments on your routes
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ada-C5/tunes-takeout/pull/1/files/6b72378a98a8c4979ad5097f4a945315962d2922#r64511396>
>